packages/react-core/src/components/DatePicker/DatePicker.tsx

```diff
@@ -40,8 +40,12 @@ export interface DatePickerProps
   inputProps?: TextInputProps;
   /** Flag indicating the date picker is disabled. */
   isDisabled?: boolean;
+  /** Flag indicating the date picker is required. */
+  isRequired?: boolean;
```

Comment:
Could you rebase.  Another PR went in to fix a different issue and it added an API for `isRequred` and `emptyDateText`